flow/raster_cache.h

```diff
@@ -42,6 +53,44 @@ class RasterCacheResult {
 
 struct PrerollContext;
 
+// The |RasterCacheImageDelegate| creates and retruns a cached result of
+// rendering the supplied |Layer| or |SkPicture|.
+class RasterCacheImageDelegate {
+ public:
+  virtual std::unique_ptr<RasterCacheResult> RasterizePicture(
```

Comment:
Docstring?

Reply:
This class was removed, but I documented the methods as they now appear in the main RasterCache class as suggested.